### DISCUSSION TITLE
h3d format file manual

### DISCUSSION THREAD

**Discussion by tangweichang-dev** [OpenRadioss/OpenRadioss#2918] [Q&A]
Hello, we want to convert the results calculated using OpenRadio on our platform into H3D format and store them in our SDM. However, the web lightweight component needs to read H3D format files. Do you have any documentation about H3D format?

**Comment by laurent-altr** [comment]
Hello, 
Can you use /ANIM instead of /H3D? It may be easier to work with /ANIM format, because it can be converted into [vtk](https://github.com/OpenRadioss/OpenRadioss/tree/main/tools/anim_to_vtk )

**Comment by elequiniou** [comment]
As explained by Laurent, H3D is an Altair output proprietary format which is not publicly disclosed. 
So best is to have a look at Anim converters and supported formats: vtk and d3plot now thanks to a recent community contribution, please have a look to: https://github.com/orgs/OpenRadioss/discussions/2790
I'll close this ticket as issue but move it to discussion. In the future HDF5 might be another interesting option.
